api/base/serializers.py

```diff
@@ -1698,3 +1698,9 @@ class MaintenanceStateSerializer(ser.ModelSerializer):
     class Meta:
         model = MaintenanceState
         fields = ('level', 'message', 'start', 'end')
+
+
+class FilterOnlyField(ser.SerializerMethodField):
+
+    def bind(self, field_name, parent):
```

Comment:
This feels like a strange implementation for this, mainly because I'm not sure why its necessary -- is there a way to accomplish a filter-only field that isn't using a subclass of `SerializerMethodField` that has no method associated with it? If this is indeed the best way, perhaps a docstring to explain why this was needed would be helpful.